packages/visual-editor/src/components/Draggable/DraggableComponent.tsx

```diff
@@ -43,67 +44,41 @@ export const DraggableComponent = ({
   style?: CSSProperties;
   isDragDisabled?: boolean;
 }) => {
-  const tooltipRef = useRef<HTMLDivElement>(null);
-  const draggableRef = useRef<HTMLDivElement>(null);
-
-  const previewSize = '100%'; // This should be based on breakpoints and added to usememo dependency array
-
-  const tooltipStyles = useMemo(() => {
-    const tooltipRect = tooltipRef.current?.getBoundingClientRect();
-    const draggableRect = draggableRef.current?.getBoundingClientRect();
-
-    const newTooltipStyles = getTooltipPositions({
-      previewSize,
-      tooltipRect,
-      coordinates: draggableRect,
-    });
-
-    return newTooltipStyles;
-
-    // Ignore eslint because we intentionally want to trigger this whenever a user clicks on a container/component which is tracked by these coordinates of the component being clicked being changed
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [coordinates]);
-
   return (
-    <div ref={draggableRef}>
-      <Draggable key={id} draggableId={id} index={index} isDragDisabled={isDragDisabled}>
```

Comment:
Mostly a note to self but sharing FYI: I might have some refactoring to do after this is merged.  In both my margin auto PR (#286) and section component poc (#279) I had moved the `className` prop to this wrapping div w/ the draggableRef.  It looks like without the outer wrapping div element, that can probably move back to the div w/ the DraggableComponent styles class

Reply:
Actually just left a comment on your PR #286 about this same thing 😅